src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/Content/PageBody/Details/Shared/_shipping.html.twig

```diff
@@ -0,0 +1,59 @@
+{% set variant = hookable_data.variant %}
+
+<div class="card-header">
+    <div class="card-title">{{ 'sylius.ui.shipping'|trans }}</div>
+</div>
+<div class="card-body">
+    <div class="divide-y">
+        <div class="row">
+            <div class="col-12 col-md-4 fw-bold">{{ 'sylius.ui.shipping_category'|trans }}:</div>
+            <div class="col-12 col-md-8">
+                {% if variant.shippingCategory is null %}
+                    <i>{{ 'sylius.ui.no_shipping_category'|trans }}</i>
+                {% else %}
+                    {{ variant.shippingCategory }}
+                {% endif %}
+            </div>
```

Comment:
Each of these information pieces should be a separate hookable. Adding/removing information would be easier thanks to it.